classroom.rb

```diff
@@ -0,0 +1,8 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+end
```

Comment:
- [ Required ] It would be better if you add a method `add_student` that can be used to add `student` to a `classroom` and also set the `classroom` for the `student`. Currently, there is no `method` in your class that can be used to do this. You can achieve this with the code below.

```ruby
  def add_student(student)
    @students << student
    student.classroom = self
  end
```
## Project requirements
![Screenshot from 2023-02-11 00-07-17](https://user-images.githubusercontent.com/16392046/218197755-85ef3134-d7ed-4093-891c-0b1f40dbf1ab.png)

Reply:
Thanks, @Felix45, I did it and it is the best way to take control of the classes 	😄